src/openrct2/network/NetworkServerAdvertiser.h

```diff
@@ -20,7 +20,7 @@
 
 #include "../common.h"
 
-enum ADVERTISE_STATUS
+enum class ADVERTISE_STATUS
 {
     ADVERTISE_STATUS_DISABLED,
     ADVERTISE_STATUS_UNREGISTERED,
```

Comment:
You can now remove `ADVERTISE_STATUS_` prefix.

Reply:
Good point. Fixed.